lib/email/sender.rb

```diff
@@ -239,6 +241,29 @@ def self.host_for(base_url)
 
     private
 
+    def add_attachments(post)
+      post.uploads.each do |upload|
+        next if FileHelper.is_supported_image?(upload.original_filename)
+        begin
+          if Discourse.store.internal?
+            file = Discourse.store.path_for(upload)
+          else
+            file = FileHelper.download(
+              "https:#{upload.url}",
```

Comment:
Are we certain that it'll always be `https`? We might want to check `SiteSetting.force_https` here.